### DISCUSSION TITLE
Server rules/guidelines

### DISCUSSION THREAD

**Discussion by object-Object** [HexxyCraft/modpack#13] [General]
Discuss rules for the Minecraft server here.

Things that've been suggested already:
* No griefing
* No pvp without consent
* Don't try to lag/break the server

**Comment by maraxin** [comment]
Some obvious ones:

- No cheating
- No duping
- No abusing bugs? (I am not sure about this one, as infinite regen 6 is considered as a bug)
- No bullying/harrasment etc.
